### PR TITLE
Add telemetry

### LINKED ISSUE #14: Add telemetry in bicep template
- Add telemetry in `main.bicep`.
- Add a parameters to be able to disable it
- Add an explanation in the Quickstart guide

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

### PR DESCRIPTION
**Pull request by aarthiem**
Update README.md

## Purpose
Adds telemetry and details to other App Templates

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Run the front-end

```
cd src/frontend
npm install
npm start
```

* Run the AI Shopping Cart Service

```
cd ../ai-shopping-cart-service
./mvnw clean package
java -jar target/ai-shopping-cart-service-<add-the-version-here>.jar
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: infra/main.bicep [hunk 1]
```diff
@@ -216,6 +216,22 @@ module aiShoppingCartService 'app/ai-shopping-cart-service.bicep' = {
     azureOpenAiDeploymentId: azureOpenAiDeploymentId
   }
 }
+//  Telemetry Deployment
+@description('Enable usage and telemetry feedback to Microsoft.')
+var telemetryId = '11d2e1bb-4e66-4a54-9d49-df3778d0e9a1-asaopenai-${location}'
+
+resource telemetrydeployment 'Microsoft.Resources/deployments@2021-04-01' = if (enableTelemetry) {
```
Comment (pmalarme) [inline infra/main.bicep:229]: Telemetry should be put in the main. The parameter enableTelemetry is missing

**Review by pmalarme — CHANGES_REQUESTED** [review]
Telemetry should be in the main. We need to add a parameter to enable or disable the telemetry

File: README.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ AI Shopping Cart is a sample application that supercharges your shopping experie
 
 > This sample application take inspiration on this original work: https://github.com/lopezleandro03/ai-assisted-groceries-cart
 
+> Refer to the [App Templates](https://github.com/microsoft/App-Templates) repo Readme for more samples that are compatible with [`azd`](https://github.com/Azure/azure-dev/).
```
Comment (pmalarme) [inline README.md:35]: We should add an explanation on telemetry in the README

Comment (aarthiem) [reply README.md:35]: added

**Review by aarthiem — COMMENTED** [review]
looks good.

**Review by pmalarme — APPROVED** [review]

**Pull request merged**